src/modules/ui/components/layout/index.js

```diff
@@ -125,6 +208,9 @@ Layout.propTypes = {
   leftPanel: React.PropTypes.func.isRequired,
   preview: React.PropTypes.func.isRequired,
   downPanel: React.PropTypes.func.isRequired,
+  stories: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+  showSearchBox: React.PropTypes.bool.isRequired,
+  actions: React.PropTypes.object.isRequired,
```

Comment:
Don't pass the whole actions here. Just define the two actions, you really use.
It allows better documentation (self).